virttest/shared/cfg/guest-os/Linux/RHEL/9/9.devel.cfg

```diff
@@ -0,0 +1,9 @@
+- devel:
+    # Uses defaults set in 9.cfg
+    # Adding x.y versions should only require changing the kickstart
+    # and adding per-arch image shas.
+    image_name = images/${os_variant}devel-${vm_arch_name}
+    unattended_install.cdrom:
+        cdrom_cd1 = isos/linux/RHEL-9-devel-${vm_arch_name}.iso
+    unattended_install, check_block_size, svirt_install:
+        unattended_file = unattended/CentOS-Stream-9-devel.ks
```

Comment:
Hi @richtja since RHEL.9 would use the same unattended configuration with CentOS-Stream.9, do we need to consider about enlarging the disk size as well?

Reply:
Hi @luckyh I didn't have problems with disk size during my testing, so I am not sure about this.

Reply:
OK, no problem now, thanks!